cdap-app-fabric/src/main/java/co/cask/cdap/app/store/RuntimeStore.java

```diff
@@ -32,6 +32,18 @@
  * A store for runtime information of a {@link Program}.
  */
 public interface RuntimeStore {
+  /**
+   * Logs initialization of program run.
+
+   * @param id id of the program
+   * @param pid run id
+   * @param startTime start timestamp in seconds
+   * @param twillRunId Twill run id
+   * @param runtimeArgs the runtime arguments for this program run
+   * @param systemArgs the system arguments for this program run
+   */
+  void setInit(ProgramId id, String pid, long startTime, @Nullable String twillRunId,
```

Comment:
`STARTING`. I agree that this is a bit confusing, because `setStart` from the store sets the status to `RUNNING`.

Should I do rename these sets of methods so that:

- `setStart` sets program's status to `ProgramRunStatus.STARTING`
- `setRunning` sets program's status to `ProgramRunStatus.RUNNING`